### ISSUE TITLE
login with 2fa  not working

### ISSUE THREAD

**Issue by Truciolo99** [joshpatten/PVE-VDIClient#86]
Hi,
I've enabled TFA in config 

auth_totp=true

but after typing username, password and OTP Key I got "Invalid username and/or password, please try again!"
![Schermata del 2023-11-30 09-03-19](https://github.com/joshpatten/PVE-VDIClient/assets/96883828/8938a0d7-e111-4b51-a83e-6cca78c832bb)

**Comment by crazychatting** [comment]
same here. how can I provide more debugging infos?

**Comment by joshpatten** [comment]
To verify, you are able to log in with this user to the Proxmox web
interface?

**Comment by crazychatting** [comment]
correct, doing this alsmost every day. TOTP works fine with realm pve and pam wie web interface

**Comment by addelovein** [comment]
Proxmox changed the API, proxmoxer code is not changed... There is an issue where ive posted the way Proxmox does the OTA with an Bash example... But no response there for weeks... Sadly since its the only Wrapper that is usable (well maybe not anymore)
Thinking of writing my own instead, but have to many projects right now... :-/

**Comment by joshpatten** [comment]
I'm going to see if I can make some progress on this today. I didn't
realize that the API changed.

**Comment by addelovein** [comment]
Since you use Proxmoxer API its in their repo you will need to do the change. 

It is the Proxmox API that changed and proxmoxer hasnt updated... Just to be clear. ;-)

**Comment by jpattWPC** [comment]
I submitted a PR: https://github.com/proxmoxer/proxmoxer/pull/158 to the proxmoxer library. I verified this worked with the VDI client.
